README.md

```diff
@@ -386,18 +386,32 @@ public CloudTasksClient getService() throws IOException {
 Long running operations (LROs) are often used for API calls that are expected to
 take a long time to complete (e.g. provisioning a GCE instance or a Dataflow pipeline).
 The initial API call creates an "operation" on the server and returns an operation ID
-to track its progress.
+to track its progress. LRO RPCs will have the suffix `Async` appended to the call name
+(i.e. `clusterControllerClient.createClusterAsync()`)
 
-Our generated gRPC clients provide a nice interface for starting the operation and
+Our generated clients provide a nice interface for starting the operation and
 then waiting for the operation to complete. This is accomplished by returning an
 [`OperationFuture`](https://cloud.google.com/java/docs/reference/gax/latest/com.google.api.gax.longrunning.OperationFuture).
 When you call `get()` on the `OperationFuture` we poll the operation endpoint to
 check on the operation. These polling operations have a default timeout that
 varies from service to service and will throw a `java.util.concurrent.CancellationException`
 with the message: `Task was cancelled.` after that timeout has been reached.
 
-### Configuring LRO Timeouts
+If you receive a `java.util.concurrent.CancellationException`, it does not mean that
+the backend GCP Operation was cancelled. This exception is thrown from the client library to let
+you know that the client library has stopped polling for the Operation's status. It respects
+the configured values in OperationTimedPollAlgorithm for each RPC. Since the LRO returns a unique
+Operation ID, you may still manually poll for the status.
```

Comment:
> you may still manually poll for the status.

Is this helpful, common thing to do? Reading "Since the LRO returns a unique Operation ID, you may still manually poll for the status.", I don't know whether I, a user, should learn more about it or not.

Reply:
Good point. I was hesitant to put this part, but it was a previous customer issue. Generally not encouraged to poll manually, but we have had people do it before as they have had specific use cases (https://github.com/googleapis/google-cloud-java/issues/9118#issuecomment-1433136433)

I'll remove this blurb as I think it would be better for customers to tinker with the LRO values instead.